docs/hub/_toctree.yml

```diff
@@ -57,8 +57,8 @@
   - local: models-libraries
     title: Integrated Libraries
     sections:
-      - local: adapter-transformers
-        title: Adapter Transformers
+      - local: adapters
```

Comment:
This would break old links pointing to hf.co/docs/hub/adapter-transformers.

The way to fix it is by crating a redirect as in https://github.com/huggingface/hub-docs/blob/main/docs/hub/_redirects.yml

Reply:
Thanks for the hint!